providers/amazon/src/airflow/providers/amazon/aws/operators/bedrock.py

```diff
@@ -654,9 +656,37 @@ def execute_complete(self, context: Context, event: dict[str, Any] | None = None
         return validated_event["ingestion_job_id"]
 
     def execute(self, context: Context) -> str:
-        ingestion_job_id = self.hook.conn.start_ingestion_job(
-            knowledgeBaseId=self.knowledge_base_id, dataSourceId=self.data_source_id
-        )["ingestionJob"]["ingestionJobId"]
+        def start_ingestion_job():
+            try:
+                ingestion_job_id = self.hook.conn.start_ingestion_job(
+                    knowledgeBaseId=self.knowledge_base_id, dataSourceId=self.data_source_id
+                )["ingestionJob"]["ingestionJobId"]
+
+                return ingestion_job_id
+            except ClientError as error:
+                error_message = error.response["Error"]["Message"].lower()
+                is_known_retryable_message = (
```

Comment:
nit. I guess this is more a question of style and personal opinion but do not you think it would be clearer/easier to read if you use `@tenacity`?

Reply:
I did consider that, but the Bedrock operators already use this pattern in other Operators, so I felt it was better to keep the pattern rather than have multiple going. What do you think?

Reply:
I think I would personally prefer tenacity but I will not block the PR because of that

Reply:
Okay, I have this tested and working and I don't want to burn too much more time on this refactoring it right now. I think I'll merge it so that we get the benefit and we can circle back and change all the operators to use tenacity later if we find the time!

Thanks for the review :smiley: